src/app/pages/administration/jobs/system-triggers-view.jsx

```diff
@@ -23,6 +26,7 @@ const headers = [
   { property: 'description', text: 'Description' },
   { property: 'nextRunDate', text: 'Next Run Date' },
   { property: 'triggerScheduleType', text: 'Trigger Schedule Type' },
+  { property: 'actions', text: 'Actions' },
```

Comment:
The "User Scheduled Jobs" table has this same Actions column, but I believe has the "invisible = true" annotation. We need the text to exist for 508 purposes, but it doesn't have to be visible on the page, and generally we don't show it

```suggestion
  { property: 'actions', text: 'Actions', invisible: true },
```